sal_interfaces/MTM1M3TS/MTM1M3TS_Events.xml

```diff
@@ -515,4 +515,86 @@
       <Count>1</Count>
     </item>
   </SALEvent>
+  <SALEvent>
+    <Subsystem>MTM1M3</Subsystem>
+    <EFDB_Topic>MTM1M3_logevent_pumpStatus</EFDB_Topic>
```

Comment:
I believe this needs to be MTM1M3TS

Reply:
You are correct. Fixed. Too many hunters - a rabbit dead, a Czech proverb.